roles/devscripts/tasks/sub_tasks/14_user.yml

```diff
@@ -49,10 +49,18 @@
 - name: Enable IP forwarding in the Network Operator.
   when:
     - cifmw_devscripts_config['network_type'] == 'OVNKubernetes'
-    - "cifmw_devscripts_config.openshift_version is ansible.builtin.version('4.14.0', '>=')"
-  ansible.builtin.copy:
-    src: "files/forwarding.yaml"
-    dest: "{{ cifmw_devscripts_config['assets_extra_folder'] }}/forwarding.yaml"
+    - cifmw_devscripts_config.openshift_version is ansible.builtin.version('4.14.0', '>=') or
+      cifmw_devscripts_host_routing
```

Comment:
cifmw_devscripts_host_routing|bool or may be can drop this condition completely as already checked in template

Reply:
I missed that one... thank you. Let's have the condition so that the config is not empty with
`OCP version is less than 4.14.0` and `routingViaHost: false`.